qiskit/circuit/library/arithmetic/polynomial_pauli_rotations.py

```diff
@@ -214,15 +214,13 @@ def degree(self) -> int:
         return 0
 
     @property
+    @deprecate_func(
```

Comment:
Oh we should remove these long-deprecated methods in the circuit library (there's another one in the `IntegerComparator` like this). Do you want to remove them in this PR or should we do it in a follow up? 🙂

Reply:
I'd rather that be done in a follow up